_includes/v2.0/metric-names.md

```diff
@@ -0,0 +1,246 @@
+Name | Help
```

Comment:
@rmloveland, this is great, but how did you get these? If possible, I'd love for us to auto-generate these in the `cockroach` repo the way we auto-generate [cluster settings](https://github.com/cockroachdb/cockroach/blob/master/docs/generated/settings/settings.html), etc. @dt for thoughts.

Reply:
@jseldess I pulled them out of the source code with `grep` and some editor macros.  Would def love to have a reproducible way to get at these tho.